### ISSUE TITLE
Date field with a defined date format loses value after submit

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#502]
**Description**
A date field with a defined date format (example: DD/MM/YYYY) loses its value after submit.
This happens on a multi-pages form, and also when the date field is setup within a group.

**Steps to reproduce**
1. Setup a form with a date field
2. Change the date field date format from YYYY-MM-DD to DD/MM/YYYY
3. Check the submission result

**Additional info**
Plugin version: 1.4.7
Craft version: 3.6.17
Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.8](https://github.com/verbb/formie/releases/tag/1.4.8)
